server/models/ToC/index.js

```diff
@@ -0,0 +1,10 @@
+import { collections } from '../../utils/mongoCollections'
+
+const ToCModel = {
+  update: async (db, query, dataFileAttributes) =>
```

Comment:
another case where we might want to name this `upsert`

Reply:
[c1cf0d0](https://github.com/AMP-SCZ/dpdash/pull/382/commits/c1cf0d04b9d9dd43b6aceef72cc4f1f45af67a9c)